packages/xrpc/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@atproto/common": "*",
```

Comment:
Might be good to get an early read on whether `@atproto/api` will still work on clients where we need it to.  Not sure how some deps in `@atproto/common` including `multiformats/cid` will play on the web, for example.

Reply:
`multiformats/cid` is totally fine on web, we used it a bunch at Fission 👍 

I think everything in here should still be playing nicely 🤔